python/hail/table.py

```diff
@@ -1450,12 +1461,13 @@ def _process_joins(self, *exprs):
         broadcast_f = lambda left, data, jt: Table(left._jt.annotateGlobalJSON(data, jt))
```

Comment:
is that comment about ordering out of date now?

Reply:
ya, it's in the wrong place.